hack/ci/check.sh

```diff
@@ -18,9 +18,58 @@
 
 set -ex
 
-./kubectl-direct_csi info
-./kubectl-direct_csi drives list
-./kubectl-direct_csi drives list | grep -q Available
-./kubectl-direct_csi drives format --all
-sleep 5
-./kubectl-direct_csi drives list
+function check_installation()
+{
+    ./kubectl-direct_csi info
+    ./kubectl-direct_csi drives list --all
+    ./kubectl-direct_csi drives format --all
+    sleep 5
+    ./kubectl-direct_csi drives list --all
+}
+
+function deploy_minio()
+{
+    kubectl apply -f hack/ci/minio.yaml
+    sleep 1m
+    kubectl get pods
+
+    runningpods=$(kubectl get pods --field-selector=status.phase=Running --no-headers | wc -l)
+    if [[ $runningpods -ne 4 ]]
+    then
+        echo "MinIO deployment failed"
+        exit 1
+    fi
+}
+
+function uninstall_minio()
+{
+    kubectl delete -f hack/ci/minio.yaml
+    sleep 1m
+    kubectl delete pvc --all
+    sleep 1m
+    ./kubectl-direct_csi volumes ls
+    ./kubectl-direct_csi drives ls --all
+
+    directcsivolumes=$(./kubectl-direct_csi volumes ls | wc -l)
+    if [[ $directcsivolumes -gt 1 ]]
+    then
+        echo "Volumes were not cleared upon deletion"
+        exit 1
+    fi
+
+    inusedrives=$(./kubectl-direct_csi drives ls | grep -q InUse | wc -l)
+    if [[ $inusedrives -gt 0 ]]
```

Comment:
`./kubectl-direct_csi drives ls | grep -q InUse && echo "disks are in use" && return 1`